packages/ckeditor5-dev-utils/tests/tools.js

```diff
@@ -577,6 +577,17 @@ describe( 'utils', () => {
 						expect( infoSpy.secondCall.args[ 0 ] ).to.match( new RegExp( files[ 1 ] ) );
 					} );
 			} );
+
+			it( 'works with paths that contain backslash', () => {
+				return tools.clean( 'test\\path', '**\\packages' )
+					.then( () => {
+						expect( delArg ).to.equal( path.join( 'test/path', '**/packages' ) );
+						expect( loggerVerbosity ).to.equal( 'info' );
+						expect( infoSpy.calledTwice ).to.equal( true );
+						expect( infoSpy.firstCall.args[ 0 ] ).to.match( new RegExp( files[ 0 ] ) );
+						expect( infoSpy.secondCall.args[ 0 ] ).to.match( new RegExp( files[ 1 ] ) );
```

Comment:
These lines are not related to the subject of this test case - you are testing here if the joined path, that is sent to the `del` package, contains only slashes, but the `infoSpy` still operates on the `files` array that does not match the provided path `'test/path/**/packages'` at all. So comparing the `infoSpy` args with the `files` when the joined path does not match them is confusing.

Reply:
Addresed